src/ClientContextInterface.php

```diff
@@ -165,7 +175,10 @@
  * @method $this hrandfield(string $key, int $count = 1, bool $withValues = false)
  * @method $this hscan($key, $cursor, array $options = null)
  * @method $this hset($key, $field, $value)
+ * @method $this hsetf(string $key, array $keyValuePairs, HSetFArguments $arguments = null)
```

Comment:
Call `keyValuePairs` `dictionary`, like in `hmset`?

Reply:
Makes sense